code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -134,6 +134,8 @@
 	pocket_storage_component_path = /datum/component/storage/concrete/pockets/shoes
 
 /obj/item/clothing/shoes/jackboots/fast
+	name = "modified jackboots"
+	desc = "Nanotrasen-issue Security combat boots for combat scenarios or combat situations. This pair seems to be modified with lighter materials."
```

Comment:
Only person who gets these is the EXO from mail tho, if there were more classes like sec and such who had some way to access em, it would make more sense to make it less NT. I'ma do the fixes tho.

Reply:
```suggestion
	name = "modified jackboots" // NSV13 - EXO mail-boots can be told apart
	desc = "Security combat boots for combat scenarios or combat situations. This pair seems to be modified with lighter materials."
```

Reply:
suggested in [98bb89c](https://github.com/BeeStation/NSV13/pull/2458/commits/98bb89c993757938cb0bea21b6e6ed411155d707)